src/server/config/schema.js

```diff
@@ -227,11 +227,7 @@ export default async () => Joi.object({
         }))
       }))
     }).default(),
-    manifestServiceUrl: Joi.when('$dev', {
-      is: true,
-      then: Joi.string().default('https://staging-dot-catalogue-dot-elastic-layer.appspot.com/v2/manifest'),
-      otherwise: Joi.string().default('https://catalogue.maps.elastic.co/v2/manifest')
-    }),
+    manifestServiceUrl: Joi.string().default('https://staging-dot-catalogue-dot-elastic-layer.appspot.com/v2/manifest'),
```

Comment:
Shouldn't this be `https://catalogue.maps.elastic.co/v2/manifest`? Or did I miss something?

Reply:
:blush: damn ok thx